app/src/main/java/org/gdg/frisbee/android/common/GdgActivity.java

```diff
@@ -172,15 +163,13 @@ public AchievementActionHandler getAchievementActionHandler() {
     protected void onActivityResult(int requestCode, int responseCode, Intent intent) {
         super.onActivityResult(requestCode, responseCode, intent);
 
-        if (PrefUtils.isSignedIn(this) && !Utils.isEmulator()) {
+        if (PrefUtils.isSignedIn(this)) {
             switch (requestCode) {
                 case RC_SIGN_IN:
                     if (responseCode == RESULT_OK) {
                         // If the error resolution was successful we should continue
                         // processing errors.
                         mSignInProgress = STATE_SIGN_IN;
-                        App.getInstance().resetOrganizer();
```

Comment:
It did not reset anything but just overwrote the organizerChecker with values from preferences.

The resetting is now properly covered for in checkOrganizer: https://github.com/gdg-x/frisbee/pull/560/files#diff-fd216eefb97ccc0dfae2b7ae369f752bR42